docs/source/nodes/setup-run-node.md

```diff
@@ -10,13 +10,18 @@ The first step is to get a server (or equivalent) which meets [the requirements
 
 ## Secure Your Server
 
-The steps that you must take to secure your server depend on your server OS and where your server is physically located. There are many articles and books about how to secure a server. Ask a search engine.
+The steps that you must take to secure your server depend on your server OS and where your server is physically located. There are many articles and books about how to secure a server. Here we just cover special considerations when securing a BigchainDB node.
 
-Here we just cover special considerations when securing a BigchainDB node.
+When you set up the firewall, here is a list of the ports where inbound traffic is expected:
 
-TODO: Special security considerations for BigchainDB nodes.
+22 - The default SSH port (TCP) <br>
+123 - The default NTP port (**UDP**) <br>
```

Comment:
Good question... I looked into it. Indeed, port 123 doesn't need to be open to all inbound traffic. A firewall rule to allow "established and related" inbound traffic is sufficient (e.g. allow responses to requests sent to external NTP servers). I updated the documentation accordingly (now moved to a new page in the Appendices).